assets/js/admin/meta-boxes-subscription.js

```diff
@@ -327,11 +327,11 @@ jQuery( function ( $ ) {
 		return false;
 	}
 
-	$( 'body.post-type-shop_subscription #post' ).on( 'submit', function () {
+	$( 'body.post-type-shop_subscription #post, body.woocommerce_page_wc-orders--shop_subscription #order' ).on( 'submit', function () {
 		if (
 			'wcs_process_renewal' ==
 			$(
-				"body.post-type-shop_subscription select[name='wc_order_action']"
+				"body.post-type-shop_subscription select[name='wc_order_action'], body.woocommerce_page_wc-orders--shop_subscription select[name='wc_order_action']"
```

Comment:
```suggestion
				'body.post-type-shop_subscription select[name="wc_order_action"], body.woocommerce_page_wc-orders--shop_subscription select[name="wc_order_action"]'
```

Nit: use single-quotes to wrap and double-quotes for attr selector for consistency.